config/gulp/build.js

```diff
@@ -85,13 +85,8 @@ gulp.task('build-uswds-if-needed', function () {
       ));
     }
 
-    //  "federalist": "npm install --dev && gulp build && fractal build",
     const sharedOpts = { stdio: 'inherit', cwd: uswdsDir };
-    return spawnP('npm', [ 'install', '--only=dev', '--ignore-scripts' ],
-                  sharedOpts)
-      .then(() => spawnP('gulp', [ 'build' ], sharedOpts))
-      .then(() => spawnP('./node_modules/.bin/fractal', [ 'build' ],
-                         sharedOpts));
+    return spawnP('npm', [ 'run', 'federalist' ], sharedOpts);
```

Comment:
@msecret, I'm confused as to why this was changed to the series of 3 commands in the first place?  What problem were you experiencing with just `npm run federalist`?  It seems to work on my end and on `federalist-staging/build`, at least... I think whatever we run, it should delegate as much as possible to an npm script in `uswds`, so that we have as loose a coupling as possible between these two repositories.  Even if that means we add yet another npm script called e.g. `npm run build-fractal-for-uswds-docs` or something nutty.